lib/gprd.gi

```diff
@@ -510,19 +510,29 @@ local iso;
     fi;
   fi;
 
-  # the ...Op is installed for `IsGroupHomomorphism'. So we have to enforce
-  # the filter to be set.
-  if not IsGroupHomomorphism(ghom) or not IsGroupHomomorphism(hhom) then
-    Error("mappings are not homomorphisms");
```

Comment:
Hmm, doesn't this mean the user will get strictly worse error messages now in case `ghom` and/or `hhom` *really* isn't a homomorphism? I.e. there will be a "no method found" error instead of this very specific information.

Reply:
I don't actually understand why it is necessary and/or beneficial to remove this check here for the purpose of this PR?